### ISSUE TITLE
comment-not-capitalized lint should ignore @todo markers

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#805]
The `comment-not-capitalized` lint flags comments that start with `@todo` as not starting with a capital letter.

For example:
```eo
# @todo #4475:35min Enable this feature after resize is implemented.
[] > some-object
```

This triggers the warning:
```
The comment doesn't start with a capital English letter (comment-not-capitalized)
```

The lint should recognize `@todo` (and similar markers like `@fixme`) as valid comment starters and skip the capitalization check for these lines, or check the first word after the marker.

**Comment by volodya-lombrozo** [comment]
Resolved by #960.
